src/BenchmarkDotNet.Core/Columns/QuantityType.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BenchmarkDotNet.Columns
+{
+    public enum UnitType
+    {
+        None,
```

Comment:
I don't like that we use `None` value for different column types. Could we introduce some additional types which help us to understand what kind of data do we use in a column? E.g. `Number`, `String`, etc?